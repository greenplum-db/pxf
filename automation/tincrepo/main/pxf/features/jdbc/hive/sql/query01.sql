-- @description query01 for JDBC Hive query without partitioning
SET datestyle TO 'ISO, MDY';
SELECT s1, s2, n1, d1, round(dc1, 18), tm, f, bg, b, tn, sml, dt, vc1, c1 FROM pxf_jdbc_hive_types_table ORDER BY s1;
