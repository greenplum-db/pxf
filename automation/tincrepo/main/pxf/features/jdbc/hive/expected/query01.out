-- @description query01 for JDBC Hive query without partitioning
SET datestyle TO 'ISO, MDY';
SELECT s1, s2, n1, d1, round(dc1, 18), tm, f, bg, b, tn, sml, dt, vc1, c1 FROM pxf_jdbc_hive_types_table ORDER BY s1;
          s1          |  s2  | n1 | d1 |            round             |         tm          |  f   |    bg    | b | tn | sml  |     dt     |  vc1  | c1  
----------------------+------+----+----+------------------------------+---------------------+------+----------+---+----+------+------------+-------+-----
 row1                 | s_6  |  1 |  6 |         1.234560000000000000 | 2013-07-13 21:00:05 |  7.7 | 23456789 | f |  1 |   10 | 2015-03-06 | abcd  | abc
 row10                | s_15 | 10 | 15 |     45678.000023400890000000 | 2013-07-22 21:00:05 |  7.7 | 23456789 | t | 10 | 1000 | 2015-03-06 | abcde | abc
 row11                | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row12_text_null      |      | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row13_int_null       | s_16 |    | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row14_double_null    | s_16 | 11 |    |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row15_decimal_null   | s_17 | 12 | 38 |                              | 2013-07-24 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row16_timestamp_null | s_16 | 11 | 37 |         0.123456789012345679 |                     |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row17_real_null      | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |      | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row18_bigint_null    | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 |          | f | 11 | 1100 | 2015-03-06 | abcde | ab 
 row19_bool_null      | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 |   | 11 | 1100 | 2015-03-06 | abcde | ab 
 row2                 | s_7  |  2 |  7 |         1.234560000000000000 | 2013-07-13 21:00:05 |  8.7 | 23456789 | t |  2 |   20 | 2015-03-06 | abcde | abc
 row20_tinyint_null   | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f |    | 1100 | 2015-03-06 | abcde | ab 
 row21_smallint_null  | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 |      | 2015-03-06 | abcde | ab 
 row22_date_null      | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 |            | abcde | ab 
 row23_varchar_null   | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 |       | ab 
 row24_char_null      | s_16 | 11 | 37 |         0.123456789012345679 | 2013-07-23 21:00:05 |  7.7 | 23456789 | f | 11 | 1100 | 2015-03-06 | abcde | 
 row3                 | s_8  |  3 |  8 |        -1.234560000000000000 | 2013-07-15 21:00:05 |  9.7 | 23456789 | f |  3 |  300 | 2015-03-06 | abcde | abc
 row4                 | s_9  |  4 |  9 | 123456789.123456789000000000 | 2013-07-16 21:00:05 | 10.7 | 23456789 | t |  4 |  400 | 2015-03-06 | abcde | abc
 row5                 | s_10 |  5 | 10 |         0.000000000001000000 | 2013-07-17 21:00:05 | 11.7 | 23456789 | f |  5 |  500 | 2015-03-06 | abcde | abc
 row6                 | s_11 |  6 | 11 |      1234.888999911100000000 | 2013-07-18 21:00:05 | 12.7 | 23456789 | t |  6 |  600 | 2015-03-06 | abcde | abc
 row7                 | s_12 |  7 | 12 |         0.000100000000000000 | 2013-07-19 21:00:05 |  7.7 | 23456789 | f |  7 |  700 | 2015-03-06 | abcde | abc
 row8                 | s_13 |  8 | 13 |     45678.000023400890000000 | 2013-07-20 21:00:05 |  7.7 | 23456789 | t |  8 |  800 | 2015-03-06 | abcde | abc
 row9                 | s_14 |  9 | 14 |     23457.100000000000000000 | 2013-07-21 21:00:05 |  7.7 | 23456789 | f |  9 |  900 | 2015-03-06 | abcde | abc
(24 rows)

